mail/getmail6/Portfile

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+github.setup        getmail6 getmail6 6.19.03 v
+
+revision            0
+categories          mail python
+platforms           {darwin any}
+maintainers         wolfsden.cz:~+macports
```

Comment:
please also add your GitHub handle and, unless there is a good reason not to, add "openmaintainer"

Reply:
You would only get notification if someone files a Trac ticket and it is assigned or Cc'd to you, or if you have your GitHub handle listed and someone submits a pull request. You won't get notification if someone commits a change to your port directly to master without a pull request.